plugins/lookup/chef_databag.py

```diff
@@ -35,6 +35,8 @@
   _raw:
     description:
       - The value from the databag
```

Comment:
```suggestion
      - The value from the databag.
```
If this is ubiquitous there, don't fix